2.0/m_geoipban.cpp

```diff
@@ -40,10 +40,23 @@ class ModuleGeoIPBan : public Module
 {
 	LocalStringExt ext;
 	GeoIP* gi;
-
+	GeoIP* gi6;
+	
 	std::string* SetExt(User* user)
 	{
-		const char* c = GeoIP_country_code_by_addr(gi, user->GetIPString());
+		const char* c = NULL;
+		
+		switch (user->client_sa.sa.sa_family)
+		{
+			case AF_INET:
+                c = GeoIP_country_code_by_addr(gi, user->GetIPString());
```

Comment:
The indentation of this (and the line below) is broken (i.e. using spaces instead of tabs).